CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add support for pre-release in Versioning.
     
 ### Fixed
+- Add a workaround to avoid 480p resolution scale down for the default video uplink policy for Chromium browsers 98 
```

Comment:
Shall we mention that we scale down the resolution to 360p instead of 480p.

Reply:
Updated.